solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -35,6 +35,8 @@ pub mod audius_data {
         let audius_admin = &mut ctx.accounts.admin;
         audius_admin.authority = authority;
         audius_admin.verifier = verifier;
+        audius_admin.initial_track_id_offset = track_id_offset; // what is this even used for?
```

Comment:
i'm guessing this initial offset is used for validation that the track we want to update is actually a legacy track. but in that case, we'd have to use the admin account for a legacy update. also, we're going to validate ownership off chain so why bother validating track id?